src/client.ts

```diff
@@ -3424,6 +3424,23 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         return this.getIgnoredUsers().includes(userId);
     }
 
+    /**
+     * Gets the list of currently ignored invites.
+     */
+    public getIgnoredInvites(): IgnoredInvites {
+        const event = this.getAccountData(IGNORED_INVITES_ACCOUNT_DATA_KEY);
+        if (!event || !event.getContent()) return {};
+        return event.getContent();
+    }
+
+    /**
+     * Update the list of ignored invites.
+     * @param invites The new list of ignored invites.
+     */
+    public setIgnoredInvites(invites: IgnoredInvites): Promise<{}> {
+        return this.setAccountData(IGNORED_INVITES_ACCOUNT_DATA_KEY, invites);
```

Comment:
Will you de duplicate the ignored rooms? Or is that the responsibility of the caller?

Reply:
I could change this, but I'm assuming that it is the responsibility of the caller.